content/changelog/2024-06-14.md

```diff
@@ -0,0 +1,82 @@
+- [Schema diff tutorial](/docs/guides/schema-diff-tutorial)
+
+### Computes, roles, and databases moved to branch pages in the Neon Console
+
+You can now find your branch's computes, roles, and databases on the branch page they belong to. This update better reflects the relationship of these objects to their specific branches in a Neon project.
+
+![new branches page](/docs/relnotes/new_branch_page.png)
+
+With this change, we also moved the compute endpoint delete option to the **Edit compute endpoint** drawer, which you can access by clicking **Edit** on the **Computes** tab.
+
+For more information about how objects in a Neon project are organized and related, see [Overview of the Neon object hierarchy](/docs/manage/overview).
+
+### Support for pgvector 0.7.1
+
+Neon now supports [pgvector](/docs/extensions/pgvector) version 0.7.1. This new version improves the performance of on-disk HNSW index builds.
+
+For the official list of updates, refer to the [pgvector changelog](https://github.com/pgvector/pgvector/blob/master/CHANGELOG.md).
+
+If you installed this extension previously and want to upgrade to the latest version, please refer to [Update an extension version](/docs/extensions/pg-extensions#update-an-extension-version) for instructions.
+
+### Integration with Outerbase
+
+We are excited to announce that the [Outerbase](https://docs.outerbase.com) integration for Neon is now publicly available.
+
+This integration enables you to instantly connect your Neon Postgres database to Outerbase’s [Data Studio](https://www.outerbase.com/products/data-studio/) and invite your team members to view, edit, query, and visualize your data.
+
+Outerbase’s [AI integration](https://www.outerbase.com/products/ai/) takes this a step further by enabling users to perform complex SQL tasks using natural language, making data manipulation accessible even to those without advanced SQL knowledge.
+
+Outerbase's' data visualization tools help you create concise and beautiful charts and dashboards, making it easier to present and interpret data.
+
+![Outerbase overview](/docs/relnotes/outerbase.gif)
+
+To learn how to connect Outerbase to your Neon project, check out [this guide](/docs/guides/outerbase).
+
+### Fixes & improvements
+
+- Fixed an issue with the dynamic rate limiter at the Neon Proxy that caused excessive CPU consumption and prevented new connections from being accepted.
+- Added a **Source** column to the **Branches** widget on the Project Dashboard showing icons that indicate the creation source for the branch. For example, you are now able to see if the branch was created in Neon, via the Neon Vercel Integration, or through our Hasura integration.
+- Added missing units of measure to the legends for several charts on the Neon **Monitoring** page in the Neon Console.
+- Added the timezone to the **Restore branch** modal. Previously, the modal only showed the date and time of the selected restore point.
```

Comment:
Add a bit more context?

Updated the **Restore branch** modal to include timezone information alongside the date and time of the selected restore point, providing clearer context for restore operations

Reply:
thank you!